src/Game/Constants.cs

```diff
@@ -92,7 +92,7 @@ internal static class Constants
         public const int MAX_STATIC_DATA_INDEX_COUNT = 0x10000;
         public const int MAX_LAND_TEXTURES_DATA_INDEX_COUNT = 0x4000;
         public const int MAX_GUMP_DATA_INDEX_COUNT = 0x10000;
```

Comment:
Technically, ClassicUO is right on this value. But if a server provides a file with more sounds, the old client does not crash - it just can't play them. Increasing this in ClassicUO makes those new sounds work. Outlands currently does this - we have extra sounds that you can only hear when playing with ClassicUO.